recipe/meta.yaml

```diff
@@ -1,26 +1,27 @@
-{% set version = "3.3" %}
+{% set version = "3.4" %}
 
 package:
   name: idna
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/i/idna/idna-{{ version }}.tar.gz
-  sha256: 9d643ff0a55b762d5cdb124b8eaa99c66322e2157b69160bc32796e824360e6d
+  sha256: 814f528e8dead7d329833b91c5faa87d60bf71824cd12a7530b5526063d02cb4
 
 build:
-  noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vvv
+  skip: True  # [py<36]
 
 requirements:
   host:
+    - flit-core
```

Comment:
Should be `flit_core >=3.2,<4`, see https://github.com/kjd/idna/blob/44fa5a0a7f7e3a5051b4d3e2ae7957c0494cd708/pyproject.toml#L2

Reply:
Added the pinning

Reply:
Sorry for misleading, the name should be `flit-core` (with a dash) not `flit_core`